src/classes/ADDR_Validator_Batch.cls

```diff
@@ -33,59 +33,90 @@
 * @group Address Management
 * @description Batch class for SmartyStreets address verification. It looks for all populated addresses and updates them if the setting is turned on.
 */
-public class ADDR_Validator_Batch implements Database.Batchable<sObject>, Database.AllowsCallouts {
+public class ADDR_Validator_Batch extends UTIL_AbstractChunkingLDV_BATCH implements Database.AllowsCallouts {
+    /**
+    * @description # of records in each batch
+    */
+    private static final Integer BATCH_SIZE = 200;
+
+    /**
+    * @description Chunk size to use when running in LDV Mode
+    */
+    private static final Integer CHUNK_SIZE = 10000000;
 
-    private String query;
+    /**
+    * @description Control whether or not to skip the already verified address 
+    */
+    private static Boolean skipPreviouslyVerified = false;
```

Comment:
This should be an instance var vs. a static var so that its value is retained as _this_ is used to chain the next batch.